languages/csharp/exercises/concept/chars/.docs/instructions.md

```diff
@@ -0,0 +1,41 @@
+In this exercise you will implement a partial set of utility routines to help a developer
+clean up identifier names.
+
+In the 4 tasks you will gradually build up the routine `Clean` A valid identifier comprises
+zero or more letters and underscores.
+
+In all cases the input string is guaranteed to be non-null. Note that the `Clean` method should treat an empty string as valid.
+
+### 1. Replace any spaces encountered with underscores
+
+This also applies to leading and trailing spaces.
+
+```csharp
+Identifier.Clean("my   Id");
+// => "my___Id"
+```
+
+### 2. Replace control characters with the upper case string "CTRL"
+
+```csharp
+Identifier.Clean("my\0Id");
+// => "myCTRLId",
+```
+
+### 3. Convert kebab-case to camel-case
+
+An identifier such as my-object becomes myObject.
```

Comment:
```suggestion

Modify the (_static_) `Identifier.Clean()` method to convert kebab-case to camelCase.

```

Maybe we should also include a link to a page that describes the casing forms? Maybe this resource: https://en.wikipedia.org/wiki/Letter_case#Special_case_styles